### ISSUE TITLE
Zero-width space added to newlines

### ISSUE THREAD

**Issue by mikesol** [deckgo/deckdeckgo#1408]
### Affected part of DeckDeckGo

- [ ] Studio (online editor)
- [ ] Remote control
- [ ] Poll
- [ ] Site
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ x] Web Components
- [ ] Templates

This report is about `highlight-code`.

### Reproduction

Steps to reproduce:

1. Go to https://mikesol.github.io/sf/crash-course and copy any of the code snippets.
2. Paste the snippet into the _blank slate_ player on [wags.fm](https://wags.fm).

### Expected Behavior

The example should be copied verbatim as it was input/written/typed (meaning what was in the code bloc provided to `deckdeckgo`).

### Actual Behavior

At least on macOS, all newlines have a zero-width space append to them.

### Environment

- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

THANK YOU for looking into this! I found a workaround to strip the zero-width spaces, but a proper fix would be helpful if possible.

**Comment by peterpeterparker** [comment]
Thanks for reporting the issue @mikesol !

The component needs zero width spaces to display empty lines, therefore, to solve your issue, I added an event listener on `copy` that checks if the action is performed within the component and cleans the spaces from the clipboard.

It has been released in the component `v3.6.0` and I also bumped the requirements in the gatsby plugin `v3.1.0`.

Let me know if it works out.
